src/CoreMedia/CMSampleBuffer.cs

```diff
@@ -1034,6 +1035,4 @@ internal CMSampleBufferAttachmentSettings (NSMutableDictionary dictionary)
 #endif
 	}
 #endif
-}
-
-#endif // !WATCH
+}
```

Comment:
Add a newline at the end of the file to avoid these funny characters (and the end of the file won't show in every future diff).